src/main/java/org/folio/circulation/support/Clients.java

```diff
@@ -412,6 +452,24 @@ private CollectionResourceClient createLoanPoliciesStorageClient(
       "/loan-policy-storage/loan-policies");
   }
 
+  private CollectionResourceClient createOverdueFinesPoliciesStorageClient(
+          OkapiHttpClient client,
+          WebContext context)
+          throws MalformedURLException {
```

Comment:
Why are these parameters on separate lines?

Reply:
To cut the length of line. Also this style is used for all methods in this class.

Reply:
Yeah, that's likely my fault. It can be corrected in a later pull request where the others are also resolved (I have them queued up and waiting)